plugins/chat/assets/javascripts/discourse/controllers/preferences-chat.js

```diff
@@ -32,8 +35,7 @@ export default class PreferencesChatController extends Controller {
   @action
   onChangeChatSound(sound) {
     if (sound && !isTesting()) {
-      const audio = new Audio(CHAT_SOUNDS[sound]);
-      audio.play();
+      this.chatAudioManager.test(sound);
```

Comment:
`test()`? In a `!isTesting()` block? That doesn't _sound_ right 😅

Reply:
Yes I had the same feeling, maybe `preview` ?

Reply:
`prehear`? :trollface:

Reply:
`sample` ?

Reply:
The alternative is some `play(sound, debounced = true)` which I'm not super fan of...

Reply:
`playImmediately()`? does what it says 😅

Reply:
well... I guess that's an acceptable tradeoff :+1:

Reply:
done